server/src/main/java/org/opensearch/index/remote/RemoteRefreshSegmentTracker.java

```diff
@@ -382,7 +414,7 @@ public void setLatestUploadedFiles(Set<String> files) {
     }
 
     private void computeBytesLag() {
-        if (latestLocalFileNameLengthMap == null || latestLocalFileNameLengthMap.isEmpty()) {
+        if (latestLocalFileNameLengthMap.isEmpty()) {
```

Comment:
Why are we removing the null check?

Reply:
Because the `latestLocalFileNameLengthMap ` field is final and has been declared with initialisation as well.
```
private final Map<String, Long> latestLocalFileNameLengthMap = ConcurrentCollections.newConcurrentMap();
```